src/Core/Style.js

```diff
@@ -371,6 +371,8 @@ class Style {
         defineStyleProperty(this, 'icon', 'key', params.icon.key);
         defineStyleProperty(this, 'icon', 'anchor', params.icon.anchor, 'center');
         defineStyleProperty(this, 'icon', 'size', params.icon.size, 1);
+        defineStyleProperty(this, 'icon', 'color', params.icon.color);
```

Comment:
Could you add documentation for these two new properties?

Reply:
done